conda/plugins/manager.py

```diff
@@ -59,10 +60,10 @@ def load_setuptools_entrypoints(self, *args, **kwargs) -> int:
         """
         try:
             return super().load_setuptools_entrypoints(*args, **kwargs)
+        except ImportError as err:
+            warnings.warn(f"WARNING: Error while loading conda plugin: {err}")
         except Exception as err:
-            raise PluginError(
-                f"Error while loading conda plugins from entrypoints: {err}"
-            )
+            raise PluginError(f"Error while loading conda plugin: {err}")
```

Comment:
Does this halt conda?

Reply:
Is the plugin hook name or dotted-name available in *args, **kwargs? Is it in the exception? Could we format that string into the error message?

Reply:
> Is the plugin hook name or dotted-name available in *args, **kwargs? Is it in the exception? Could we format that string into the error message?

No, the name of the entrypoint we're loading is just `conda` and the values passed is just that. We might be able to do more, but would implement more of the feature on our own than to rely on what is there in pluggy. I think it's a good idea since right now, it's hard for users to identify what's causing the loading error.

Reply:
> Does this halt conda?

Yeah, `PluginError` is a subclass of `CondaError` and will be gracefully handled like the other subclasses.

Reply:
So if you have a plugin that raises another exception besides ImportError, you will still have to manually delete it, or use a different conda to update that environment before you can do anything with conda?

Reply:
Are you suggesting catching all exceptions when loading plugins from 3rd party packages and printing a warning instead?

Reply:
Yes, I think that might be useful.